lib/metasploit/framework/command/console.rb

```diff
@@ -39,12 +39,50 @@ def spinner
     end
   end
 
+  def start_profiler
+    if ENV['METASPLOIT_CPU_PROFILE']
+      gem 'perftools.rb'
```

Comment:
Do we want to add perftools.rb to the Gemfile?

```
perftools.rb is not part of the bundle. Add it to your Gemfile.local
```

It doesn't seem to compile for mac + ruby 2.6.5 though 🤔

Reply:
After poking further, this issue might have existed for a few years - https://github.com/tmm1/perftools.rb/issues/78 and the project might be abandoned?

I'll try swapping this out for stackprof 👀